src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/SettingsEditSettingsTwoFactorBackupCodesDialog.tsx

```diff
@@ -26,19 +27,26 @@ const SettingsEditSettingsTwoFactorBackupCodesDialog: FC<SettingsEditSettingsTwo
       </Text>
 
       {me?.backupSecondFactors ? (
-        <GridColumns mt={2}>
-          {me.backupSecondFactors.map(factor => {
-            if (!factor) return null
+        <>
+          <GridColumns mt={2}>
+            {me.backupSecondFactors.map(factor => {
+              if (!factor) return null
 
-            return (
-              <Column span={6} key={factor.code}>
-                <Text variant="lg" textAlign="center">
-                  {factor.code}
-                </Text>
-              </Column>
-            )
-          })}
-        </GridColumns>
+              return (
+                <Column span={6} key={factor.code}>
+                  <Text variant="lg" textAlign="center">
+                    {factor.code}
+                  </Text>
+                </Column>
+              )
+            })}
+          </GridColumns>
+          <SettingsEditSettingsTwoFactorBackupCodesActions
+            backupSecondFactors={me.backupSecondFactors!.map(factor =>
+              factor!.code!.toString()
+            )}
+          ></SettingsEditSettingsTwoFactorBackupCodesActions>
```

Comment:
Tiny nit -- this `SettingsEditSettingsTwoFactorBackupCodesActions` can be self-closing

Reply:
Ah. Good catch. Fixed.